fake_spectra/spectra.py

```diff
@@ -229,27 +240,29 @@ def __init__(self,num, base,cofm, axis, res=None, cdir=None, savefile="spectra.h
     def save_file(self):
         """
         Saves spectra to a file, because they are slow to generate.
-        File is by default to be $snap_dir/snapdir_$snapnum/spectra.hdf5.
+        File is by default to be $snap_dir/snapdir_$snapnum/spectra_rank.hdf5.
```

Comment:
This comment I think is out of date? It is saved to spectra.hdf5 but only by rank 0.